### ISSUE TITLE
[Accessibility]A11y_VSCode_ScreenReaderOptimized_ColorContrast:ColorContrast ratio is less than 4.5:1 for the contents present on the screen for solarized light theme.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#52049]
**Environment Details**:
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
ColorContrast,MAS1.4.3
**Repro Steps:**
1. Launch VS Code.
2. Navigate to File->select New File->Navigate to Status Bar->Click on "Screen Reader Optimized" button.
3. Open Color Contrast analyzer and check the ratio.
**Actual:**
Color Contrast ratio is less than 4.5:1 for solarized light theme.
Same issue observed on some other theme also.
**Expected:**
Color Contrast ratio should be more than 4.5:1 for solarized light theme.
Recommendations:
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](url)
**User Impact:**
The low vision user will not able to see the content and the controls.
**MAS Reference:**
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={a73546c7-765f-489c-b18f-afb659fe99e6}](url)
Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference**

![accessibility a11y_vscode_screenreaderoptimized_colorcontrast1](https://user-images.githubusercontent.com/39204909/41471634-d17559a6-70d1-11e8-9f4f-fa0e2c23634c.png)
![accessibility a11y_vscode_screenreaderoptimized_colorcontrast2](https://user-images.githubusercontent.com/39204909/41471635-d1b1044c-70d1-11e8-8ffa-0d67f01f05d7.png)
![accessibility a11y_vscode_screenreaderoptimized_colorcontrast3](https://user-images.githubusercontent.com/39204909/41471636-d1e97dc2-70d1-11e8-8248-f8e853e08667.png)
![accessibility a11y_vscode_screenreaderoptimized_colorcontrast4](https://user-images.githubusercontent.com/39204909/41471637-d2230ede-70d1-11e8-96dc-2155159d2bd7.png)
![accessibility a11y_vscode_screenreaderoptimized_colorcontrast5](https://user-images.githubusercontent.com/39204909/41471640-d25712e2-70d1-11e8-8577-f7ade5cb9e90.png)
